api/nakadi-event-bus-api.yaml

```diff
@@ -1083,6 +1235,60 @@ definitions:
         items:
           $ref: '#/definitions/Event'
 
+  Subscription:
+    description: |
+      Subscription is a high level consumption unit. Subscriptions allow applications to easily scale the number of
+      clients by managing consumed event offsets and distributing load between instances.
+      The key properties that identify subscription are 'owning_application', 'event_types' and 'use_case'.
+      It's not possible to have two different subscriptions with these properties being the same.
+    properties:
+      id:
+        type: string
+        description: |
+          Id of subscription that was created. Is generated by Nakadi, should not be specified when creating
+          subscription.
+      created_at:
```

Comment:
You can declare properties as `readOnly: true`, that should cover your use case.

https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#fixed-fields-13